### ISSUE TITLE
Failed to install provider version 0.2.7

### ISSUE THREAD

**Issue by dzeediscoveryzone** [megaport/terraform-provider-megaport#33]
Hi All,

We're currently onboarding Megaport and deploying via terraform however I found that with the latest release (`0.2.7`) there are issues with the checksum hashes. Locking the version to `0.2.6` works as expected, the error is below:

```sh
╷
│ Error: Failed to install provider
│ 
│ Error while installing megaport/megaport v0.2.7: checksum list has
│ unexpected SHA-256 hash
│ 868fe9a60dfe7dcc0e6988bbca37952b27a10bc92cb23fa36cc3fa86d6f8aac0 (expected
│ b9a52a0130a1c4b26e2cda809d01b9a781ad3ff415094e104708cfeaaeba6619)
```

**Comment by grayson-denny-ahead** [comment]
Getting same error on multiple versions.

**Comment by flackon** [comment]
It looks like the checksum hashes between the registry and github are out of sync for 0.2.7, this will be fixed in 0.2.8.
